src/app/pages/question/question.component.ts

```diff
@@ -133,6 +134,11 @@ export class QuestionComponent implements OnInit, CanDeactivate<CanComponentDeac
     const currentIndex = this.currentIndex$.value;
     if (currentIndex > 0) {
       this.currentIndex$.next(currentIndex - 1);
+    } else {
+      this.isNotification$.next(true);
+      setTimeout(() => {
+        this.isNotification$.next(false);
```

Comment:
What is the purpose of this notification? if your notification is an error notification, let's rename it properly. And instead of isNotification$ use isError$.

Reply:
this component will show if user don't select an option and want to go to next question

Reply:
I get this recommendation from the demo and want to implement it

Reply:
I see, can it be checked by control/form?
For example, add Validators.reqired, and on the radio level add this error message with default but customisable text 'Field is required'.
And on the question component level when user clicks, check if the form.invalid, mark it as touched to fire the error. Something like that

Reply:
it's great idea, I'll implement it to my code